### ISSUE TITLE
[G3] Add test component with all ciphers and AEADs accelerated only

### ISSUE THREAD

**Issue by mpg** [Mbed-TLS/mbedtls#6316]
- Add a test component that accelerates all ciphers, modes and AEADs, with as many things enabled as currently possible
- Add a reference component with a similar config and everything built-in, and coverage comparison in `analyze_outcome.py` as usual.

For this task, it is acceptable to disable as many modules as needed initially to make things build without built-in implementations of ciphers / modes / AEADs.  Adding support for new things is out of scope here, but small bugs can be fixed immediately if easy enough. It is also acceptable to ignore as much as necessary in the coverage comparison for now.

All gaps identified here will be either decided out-of-scope for this EPIC's goals, or closed by subsequent tasks to be created.

If bugs are fixed, the fixes should be backported if the bugs were also present in 2.28.

Depends on #8064

**Comment by mpg** [comment]
Note: might want to have CMAC accelerated as well, see #6317
